code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -935,6 +935,86 @@
 	new /obj/item/ammo_box/magazine/enforcer/lethal(src)
 	new /obj/item/ammo_box/magazine/enforcer/lethal(src)
 
+/obj/item/storage/box/hydroponics_starter
+	name = "hydroponics starter kit"
+	desc = "Everything you need to start your own botany lab."
+	icon_state = "ert_box"
+
+/obj/item/storage/box/hydroponics_starter/populate_contents()
+	for(var/I in 1 to 2)
+		new /obj/item/circuitboard/hydroponics(src)
+		new /obj/item/stock_parts/matter_bin(src)
+		new /obj/item/stock_parts/matter_bin(src)
+		new /obj/item/stock_parts/manipulator(src)
+	new /obj/item/reagent_containers/glass/bucket(src)
+
+/obj/item/storage/box/turbine_kit
+	name = "turbine kit"
+	desc = "Somehow, they managed to fit almost an entire turbine assembly into this box."
```

Comment:
its just the circuitboards and a few parts, i dont really get how that would be difficult, its only 7 small items which you can normally fit in a box.

Reply:
The stock parts were meant to be a set of 6 each, since that's what the turbine requires. This exposed a different issue, which should now be resolved, and explains the "somehow".